packages/flutter/lib/src/material/user_accounts_drawer_header.dart

```diff
@@ -146,6 +147,11 @@ class UserAccountsDrawerHeader extends StatefulWidget {
   /// with its background color set to the current theme's primaryColor is used.
   final Decoration decoration;
 
+  /// The margin around the drawer header.
+  ///
+  /// If not specified, the [DrawerHeader] default is used.
```

Comment:
I think this is not quite right, if not specified a margin of null (0) is used. I will fix and write a test.